deposit/src/test/java/edu/unc/lib/deposit/CleanupDepositJobTest.java

```diff
@@ -105,6 +109,9 @@ public void setup() throws Exception {
 
 		// save mappings
 		FileUtils.writeStringToFile(modifiedStagesConfig, stages.getLocalConfig());
+		
+		PIDs.setRepository(repo);
+		when(repo.getFedoraBase()).thenReturn("http://www.fcrepo.com");
```

Comment:
Could you add a trailing / to uri?  Also, it might be helpful to make an AbstractDepositJobTest which sets this sort of thing up rather than having to do it in each test.  AbstractNormalizationJobTest could inherit from that